src/TSMapEditor/Initialization/MapLoader.cs

```diff
@@ -673,10 +672,10 @@ public static void ReadWaypoints(IMap map, IniFile mapIni)
                     AddMapLoadError($"Waypoint {waypoint.Identifier} at {waypoint.Position} is not within the valid map area.");
                     continue;
                 }
-                
-                if (mapCell.Waypoint != null)
+
+                if (mapCell.Waypoints.Count > 0)
                 {
-                    AddMapLoadError($"Cell at {waypoint.Position} has multiple waypoints placed on it. Skipping adding waypoint #{waypoint.Identifier} there.");
+                    Logger.Log($"NOTE: Waypoint {waypoint.Identifier} exists in the cell {waypoint.Position} that already has other waypoints.");
```

Comment:
Can improve this a bit (both grammar and accuracy wise).
```suggestion
                    Logger.Log($"NOTE: Waypoint {waypoint.Identifier} exists in a cell {waypoint.Position} that already has one or more waypoints.");
```